azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/ImageType.java

```diff
@@ -6,8 +6,9 @@
 
 package com.microsoft.azure.maven.webapp.configuration;
 
-public enum DockerImageType {
+public enum ImageType {
     NONE,
+    BUILT_IN,
```

Comment:
Please put the new entry at the end of the enumeration. This will avoid changing all the values of other members.